lib/src/notification.dart

```diff
@@ -102,7 +149,6 @@ AndroidSettings? _androidSettings;
 final _activeNotifications = <String, List<String>>{};
 int _nextId = 0;
 final _showIdFromNotificationId = <String, int>{};
-final _receivePort = ReceivePort();
```

Comment:
By removing the port, if/when we want to implement handlers for clicking on a notification, how do we communicate with the main app?

Reply:
I've had a look at `flutter_local_notifications` and I see that the `onDidReceiveNotificationResponse` handler is called in the main isolate. Also there is the `getNotificationAppLaunchDetails` method that can be used to get info when a notification results in an app being launched. 

So I don't see us needing to use the port again.